brigade-controller/cmd/brigade-controller/controller/controller_test.go

```diff
@@ -75,4 +93,173 @@ func TestController(t *testing.T) {
 	if c.VolumeMounts[0].Name != volumeName {
 		t.Error("Container.VolumeMounts is not correct")
 	}
+
+	if l := len(pod.Spec.InitContainers); l != 1 {
+		t.Fatalf("Expected 1 init container, got %d", l)
+	}
+	ic := pod.Spec.InitContainers[0]
+	if envlen := len(ic.Env); envlen != 5 {
+		t.Errorf("expected 5 env vars, got %d", envlen)
+	}
+
+	if ic.Image != sidecarImage {
+		t.Errorf("expected sidecar %q, got %q", sidecarImage, ic.Image)
+	}
+
+	if ic.VolumeMounts[0].Name != sidecarVolumeName {
+		t.Errorf("expected sidecar volume %q, got %q", sidecarVolumeName, ic.VolumeMounts[0].Name)
+	}
+}
+
+func TestController_WithScript(t *testing.T) {
+	// t.Skip("make better")
```

Comment:
is this made better already?

Reply:
Isn't it better just by commenting it out?